lib/Models/Subscriber.php

```diff
@@ -79,6 +79,10 @@ function isWPUser() {
     return ($this->wp_user_id !== null);
   }
 
+  function isWooCommerceUser() {
+    return $this->is_woocommerce_user == true;
```

Comment:
Usage of weak comparison `==` is kind of suspicious. I'd suggest using `return (bool) $this->is_woocommerce_user`.

Reply:
Fixed.